test/parallel/test-http-mutable-headers.js

```diff
@@ -43,8 +43,9 @@ const s = http.createServer(common.mustCall((req, res) => {
   switch (test) {
     case 'headers':
       // Check that header-related functions work before setting any headers
-      // eslint-disable-next-line no-restricted-properties
-      assert.deepEqual(res.getHeaders(), {});
+      const headers = res.getHeaders();
+      const exoticObj = Object.create(null);
```

Comment:
Nit: I am being extremely pedantic here. So please ignore this comment, if you don't agree. Quoting [9.1 Ordinary Object Internal Methods and Internal Slots](http://www.ecma-international.org/ecma-262/6.0/#sec-ordinary-object-internal-methods-and-internal-slots),

> All ordinary objects have an internal slot called [[Prototype]]. The value of this internal slot is either null or an object and is used for implementing inheritance.

This object doesn't do or posses anything *exotic*. So this is still a normal object I believe.